jekyll/_cci2_ja/troubleshooting-common-issues.md

```diff
@@ -0,0 +1,1635 @@
+{: #as-an-admin-can-i-see-the-content-of-a-dlc-volume }
+{:.no_toc}
+
+While you can see what volume is used for each job, at this time, the content is unavailable.
+
+</details>
+
+
+<details markdown=block>
+<summary>キャッシュされた Docker レイヤーを削除するには？</summary>
+
+{% raw %}
+Running `docker images --no-trunc --format '{{.ID}}' | xargs docker rmi** or **docker volume prune -f` will delete all of the images and their layers from the volume connected to your job.
+{% endraw %}
+
+
+Because of [how DLC works](https://circleci.com/docs/2.0/docker-layer-caching/#how-dlc-works) you might need to leave this command in your config and run several jobs to remove the DLC layers from all volumes associated with your project.
+
+Alternatively, you can use the config below to run a very short job that consumes all 50 DLC volumes and purges docker caches. You can push it to a feature branch without disrupting your main branches:
+
+```yml
+version: 2  
+ jobs:    
+  docker-purge:  
+    docker:    
+      - image: cimg/base:2020.01    
+    parallelism: 50  
+    steps:  
+      - setup_remote_docker  
+      - run: docker volume prune -f
+```
+
+If you're running into issues where stale volume caches are causing problems often, then consider using our new [docker registry image cache orb](https://circleci.com/orbs/registry/orb/cci-x/docker-registry-image-cache) as a substitute. It's more suitable for projects that have high concurrency throughput, or jobs that use parallelism due to the limited number of volumes available.
+
+If you notice errors always happen on the same volume, which you can see by checking the "spin up environment" step or "remote docker" please [contact support ](https://support.circleci.com/hc/en-us/requests/new?ticket%5Fform%5Fid=855268)
+
+</details>
+
+
+<details markdown=block>
+<summary>「Error response from daemon: OCI runtime create failed」 エラーを解決する方法</summary>
+
+ジョブでセカンダリ サービス コンテナを追加した際、以下のようなエラーが発生する場合があります。
+
+```
+Unexpected environment preparation error: Error response from daemon:   
+OCI runtime create failed: container_linux.go:345:   
+starting container process caused "process_linux.go:303:   
+getting the final child's pid from pipe caused \"EOF\"": unknown
+```
+
+This usually occurs due to a bad [command](https://circleci.com/docs/2.0/configuration-reference/#docker) being passed to one of the secondary service containers in your job.
+
+このエラー メッセージの原因は、セカンダリ サービス コンテナがクラッシュしたか、予期せず終了したことです。 これは、CircleCI が Docker デーモンから PID を取得できず、エラーが発生して終了したことを意味します。
+
+コンテナを正常にスピンアップするには、コマンドとエントリポイントが正しいことを確認してください。 [CircleCI CLI](https://circleci.com/docs/ja/2.0/local-cli/#%E3%83%9E%E3%82%B7%E3%83%B3%E4%B8%8A%E3%81%AE%E3%82%B3%E3%83%B3%E3%83%86%E3%83%8A%E5%86%85%E3%81%A7%E3%81%AE%E3%82%B8%E3%83%A7%E3%83%96%E3%81%AE%E5%AE%9F%E8%A1%8C) を使用すると、ローカルでのテストにより、ジョブをローカルで実行してすべての Docker コンテナが正常にスピンアップするかどうかを確認できます。 エラーを解決するには、お使いの設定ファイルで定義している Docker イメージのドキュメントを参照して、スピンアップ時に使用できるコマンドを確認してください。
```

Comment:
```suggestion
コンテナを正常にスピンアップするには、コマンドとエントリポイントが正しいことを確認してください。 [CircleCI CLI](https://circleci.com/docs/ja/2.0/local-cli/#run-a-job-in-a-container-on-your-machine) を使用すると、ローカルでのテストにより、ジョブをローカルで実行してすべての Docker コンテナが正常にスピンアップするかどうかを確認できます。 エラーを解決するには、お使いの設定ファイルで定義している Docker イメージのドキュメントを参照して、スピンアップ時に使用できるコマンドを確認してください。
```

Found the right link @ https://circleci.com/docs/ja/2.0/local-cli/#run-a-job-in-a-container-on-your-machine 🤖